Submitted By: Martin Ward <macros_the_black at ntlworld dot com>
Date: 2013-06-18
Initial Package Version: 1.0.1e
Upstream Status: Unknown
Origin: self, based on fedora
Description: Fixes install with perl-5.18.

--- doc/apps/cms.pod	2013-06-06 14:35:15.867871879 +0100
+++ doc/apps/cms.pod	2013-06-06 14:35:25.791747119 +0100
@@ -450,28 +450,28 @@
 
 =over 4
 
-=item 0
+=item C<0>
 
 the operation was completely successfully.
 
-=item 1 
+=item C<1>
 
 an error occurred parsing the command options.
 
-=item 2
+=item C<2>
 
 one of the input files could not be read.
 
-=item 3
+=item C<3>
 
 an error occurred creating the CMS file or when reading the MIME
 message.
 
-=item 4
+=item C<4>
 
 an error occurred decrypting or verifying the message.
 
-=item 5
+=item C<5>
 
 the message was verified correctly but an error occurred writing out
 the signers certificates.
--- doc/apps/smime.pod	2013-06-06 14:35:15.867871879 +0100
+++ doc/apps/smime.pod	2013-06-06 14:35:25.794747082 +0100
@@ -308,28 +308,28 @@
 
 =over 4
 
-=item 0
+=item C<0>
 
 the operation was completely successfully.
 
-=item 1 
+=item C<1>
 
 an error occurred parsing the command options.
 
-=item 2
+=item C<2>
 
 one of the input files could not be read.
 
-=item 3
+=item C<3>
 
 an error occurred creating the PKCS#7 file or when reading the MIME
 message.
 
-=item 4
+=item C<4>
 
 an error occurred decrypting or verifying the message.
 
-=item 5
+=item C<5>
 
 the message was verified correctly but an error occurred writing out
 the signers certificates.
--- doc/crypto/X509_STORE_CTX_get_error.pod	2013-06-06 14:35:15.874871791 +0100
+++ doc/crypto/X509_STORE_CTX_get_error.pod	2013-06-06 14:37:13.826388940 +0100
@@ -278,6 +278,8 @@
 an application specific error. This will never be returned unless explicitly
 set by an application.
 
+=back
+
 =head1 NOTES
 
 The above functions should be used instead of directly referencing the fields
--- doc/ssl/SSL_accept.pod	2013-06-06 14:35:15.871871829 +0100
+++ doc/ssl/SSL_accept.pod	2013-06-06 14:35:25.796747057 +0100
@@ -44,12 +44,12 @@
 
 =over 4
 
-=item 1
+=item C<1>
 
 The TLS/SSL handshake was successfully completed, a TLS/SSL connection has been
 established.
 
-=item 0
+=item C<0>
 
 The TLS/SSL handshake was not successful but was shut down controlled and
 by the specifications of the TLS/SSL protocol. Call SSL_get_error() with the
--- doc/ssl/SSL_clear.pod	2013-06-06 14:35:15.871871829 +0100
+++ doc/ssl/SSL_clear.pod	2013-06-06 14:35:25.803746969 +0100
@@ -56,12 +56,12 @@
 
 =over 4
 
-=item 0
+=item C<0>
 
 The SSL_clear() operation could not be performed. Check the error stack to
 find out the reason.
 
-=item 1
+=item C<1>
 
 The SSL_clear() operation was successful.
 
--- doc/ssl/SSL_COMP_add_compression_method.pod	2013-06-06 14:35:15.870871842 +0100
+++ doc/ssl/SSL_COMP_add_compression_method.pod	2013-06-06 14:35:25.806746931 +0100
@@ -53,11 +53,11 @@
 
 =over 4
 
-=item 0
+=item C<0>
 
 The operation succeeded.
 
-=item 1
+=item C<1>
 
 The operation failed. Check the error queue to find out the reason.
 
--- doc/ssl/SSL_connect.pod	2013-06-06 14:35:15.869871854 +0100
+++ doc/ssl/SSL_connect.pod	2013-06-06 14:35:25.808746906 +0100
@@ -41,12 +41,12 @@
 
 =over 4
 
-=item 1
+=item C<1>
 
 The TLS/SSL handshake was successfully completed, a TLS/SSL connection has been
 established.
 
-=item 0
+=item C<0>
 
 The TLS/SSL handshake was not successful but was shut down controlled and
 by the specifications of the TLS/SSL protocol. Call SSL_get_error() with the
--- doc/ssl/SSL_CTX_add_session.pod	2013-06-06 14:35:15.871871829 +0100
+++ doc/ssl/SSL_CTX_add_session.pod	2013-06-06 14:35:25.816746805 +0100
@@ -52,13 +52,13 @@
 
 =over 4
 
-=item 0
+=item C<0>
 
  The operation failed. In case of the add operation, it was tried to add
  the same (identical) session twice. In case of the remove operation, the
  session was not found in the cache.
 
-=item 1
+=item C<1>
  
  The operation succeeded.
 
--- doc/ssl/SSL_CTX_load_verify_locations.pod	2013-06-06 14:35:15.870871842 +0100
+++ doc/ssl/SSL_CTX_load_verify_locations.pod	2013-06-06 14:35:25.818746780 +0100
@@ -100,13 +100,13 @@
 
 =over 4
 
-=item 0
+=item C<0>
 
 The operation failed because B<CAfile> and B<CApath> are NULL or the
 processing at one of the locations specified failed. Check the error
 stack to find out the reason.
 
-=item 1
+=item C<1>
 
 The operation succeeded.
 
--- doc/ssl/SSL_CTX_set_client_CA_list.pod	2013-06-06 14:35:15.871871829 +0100
+++ doc/ssl/SSL_CTX_set_client_CA_list.pod	2013-06-06 14:35:25.821746742 +0100
@@ -66,11 +66,11 @@
 
 =over 4
 
-=item 1
+=item C<1>
 
 The operation succeeded.
 
-=item 0
+=item C<0>
 
 A failure while manipulating the STACK_OF(X509_NAME) object occurred or
 the X509_NAME could not be extracted from B<cacert>. Check the error stack
--- doc/ssl/SSL_CTX_set_session_id_context.pod	2013-06-06 14:35:15.871871829 +0100
+++ doc/ssl/SSL_CTX_set_session_id_context.pod	2013-06-06 14:35:25.828746654 +0100
@@ -64,13 +64,13 @@
 
 =over 4
 
-=item 0
+=item C<0>
 
 The length B<sid_ctx_len> of the session id context B<sid_ctx> exceeded
 the maximum allowed length of B<SSL_MAX_SSL_SESSION_ID_LENGTH>. The error
 is logged to the error stack.
 
-=item 1
+=item C<1>
 
 The operation succeeded.
 
--- doc/ssl/SSL_CTX_set_ssl_version.pod	2013-06-06 14:35:15.871871829 +0100
+++ doc/ssl/SSL_CTX_set_ssl_version.pod	2013-06-06 14:35:25.831746617 +0100
@@ -42,11 +42,11 @@
 
 =over 4
 
-=item 0
+=item C<0>
 
 The new choice failed, check the error stack to find out the reason.
 
-=item 1
+=item C<1>
 
 The operation succeeded.
 
--- doc/ssl/SSL_CTX_use_psk_identity_hint.pod	2013-06-06 14:35:15.870871842 +0100
+++ doc/ssl/SSL_CTX_use_psk_identity_hint.pod	2013-06-06 14:36:42.456783309 +0100
@@ -81,6 +81,8 @@
 
 Return values from the server callback are interpreted as follows:
 
+=over
+
 =item > 0
 
 PSK identity was found and the server callback has provided the PSK
@@ -94,9 +96,11 @@
 connection will fail with decryption_error before it will be finished
 completely.
 
-=item 0
+=item C<0>
 
 PSK identity was not found. An "unknown_psk_identity" alert message
 will be sent and the connection setup fails.
 
+=back
+
 =cut
--- doc/ssl/SSL_do_handshake.pod	2013-06-06 14:35:15.869871854 +0100
+++ doc/ssl/SSL_do_handshake.pod	2013-06-06 14:35:25.839746516 +0100
@@ -45,12 +45,12 @@
 
 =over 4
 
-=item 1
+=item C<1>
 
 The TLS/SSL handshake was successfully completed, a TLS/SSL connection has been
 established.
 
-=item 0
+=item C<0>
 
 The TLS/SSL handshake was not successful but was shut down controlled and
 by the specifications of the TLS/SSL protocol. Call SSL_get_error() with the
--- doc/ssl/SSL_read.pod	2013-06-06 14:35:15.871871829 +0100
+++ doc/ssl/SSL_read.pod	2013-06-06 14:35:25.847746415 +0100
@@ -86,7 +86,7 @@
 The read operation was successful; the return value is the number of
 bytes actually read from the TLS/SSL connection.
 
-=item 0
+=item C<0>
 
 The read operation was not successful. The reason may either be a clean
 shutdown due to a "close notify" alert sent by the peer (in which case
--- doc/ssl/SSL_session_reused.pod	2013-06-06 14:35:15.871871829 +0100
+++ doc/ssl/SSL_session_reused.pod	2013-06-06 14:35:25.849746390 +0100
@@ -27,11 +27,11 @@
 
 =over 4
 
-=item 0
+=item C<0>
 
 A new session was negotiated.
 
-=item 1
+=item C<1>
 
 A session was reused.
 
--- doc/ssl/SSL_set_fd.pod	2013-06-06 14:35:15.869871854 +0100
+++ doc/ssl/SSL_set_fd.pod	2013-06-06 14:35:25.852746353 +0100
@@ -35,11 +35,11 @@
 
 =over 4
 
-=item 0
+=item C<0>
 
 The operation failed. Check the error stack to find out why.
 
-=item 1
+=item C<1>
 
 The operation succeeded.
 
--- doc/ssl/SSL_set_session.pod	2013-06-06 14:35:15.870871842 +0100
+++ doc/ssl/SSL_set_session.pod	2013-06-06 14:35:25.855746315 +0100
@@ -37,11 +37,11 @@
 
 =over 4
 
-=item 0
+=item C<0>
 
 The operation failed; check the error stack to find out the reason.
 
-=item 1
+=item C<1>
 
 The operation succeeded.
 
--- doc/ssl/SSL_shutdown.pod	2013-06-06 14:35:15.870871842 +0100
+++ doc/ssl/SSL_shutdown.pod	2013-06-06 14:35:25.857746290 +0100
@@ -92,12 +92,12 @@
 
 =over 4
 
-=item 1
+=item C<1>
 
 The shutdown was successfully completed. The "close notify" alert was sent
 and the peer's "close notify" alert was received.
 
-=item 0
+=item C<0>
 
 The shutdown is not yet finished. Call SSL_shutdown() for a second time,
 if a bidirectional shutdown shall be performed.
--- doc/ssl/SSL_write.pod	2013-06-06 14:35:15.870871842 +0100
+++ doc/ssl/SSL_write.pod	2013-06-06 14:35:25.865746189 +0100
@@ -79,7 +79,7 @@
 The write operation was successful, the return value is the number of
 bytes actually written to the TLS/SSL connection.
 
-=item 0
+=item C<0>
 
 The write operation was not successful. Probably the underlying connection
 was closed. Call SSL_get_error() with the return value B<ret> to find out,
